python/orca/src/bigdl/orca/learn/tf/estimator.py

```diff
@@ -607,10 +661,13 @@ def fit(self, data,
                                    checkpoint_trigger=checkpoint_trigger)
         return self
 
-    def predict(self, data, batch_size=4,
-                feature_cols=None,
-                auto_shard_files=False,
-                ):
+    def predict(
+        self,
+        data: Union["SparkXShards", "DataFrame", "tf.data.Dataset"],
```

Comment:
Need to double check if predict supports `tf.data.Dataset`, maybe just comments mentioned.

Reply:
Deleted.